core/templates/pages/exploration-editor-page/modal-templates/post-publish-modal.component.ts

```diff
@@ -67,8 +70,10 @@ export class PostPublishModalComponent
     range.setStartBefore((codeDiv as HTMLDivElement).firstChild as Node);
     range.setEndAfter((codeDiv as HTMLDivElement).lastChild as Node);
     let selection = window.getSelection();
-    selection.removeAllRanges();
-    selection.addRange(range);
+    if (selection !== null) {
```

Comment:
Is it conceptually fine when `selection` is `null`? Should we maybe throw an error if it is `null`?

Reply:
Done.